.github/workflows/build.yaml

```
@@ -5,17 +5,16 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
-    env:
-      # disable the demon, as it's causing some problems
-      GRADLE_OPTS: -Dorg.gradle.daemon=false
-
+    strategy:
+      matrix:
+        version: [ 8, 9, 11 ]
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up JDK 1.8
+      - name: Set up JDK
```

Is it legal to use dynamic expressions in step names? So this would become something like

```suggestion
      - name: Set up JDK version ${{ matrix.version }}
```

?

---

Let's find out!